code/modules/gunse/modular_gun_parts.dm

```diff
@@ -429,6 +429,63 @@ ABSTRACT_TYPE(/obj/item/gun_parts/accessory)
 	icon_state = "it_revolver_short"
 	length = 13
 
+
+/obj/item/gun_parts/barrel/soviet/improvshort
+	name = "Improvised Short Barrel"
+	desc = "Some pipe shrouded with the remains of a soviet spacesuit"
+	spread_angle = -2
+	jam_frequency_fire = -3
```

Comment:
not sure anything should have a negative jam modifier.

Reply:
Forgot to remove the - sign. No clue where it came from though. upped the jam by one